ansible/roles/etc_hosts/README.md

```diff
@@ -1,7 +1,11 @@
 # etc_hosts
 
-Hosts in the `etc_hosts` groups have `/etc/hosts` created. The generated file defines all hosts in this group using `ansible_host` as the IP address and `inventory_hostname` as the canonical hostname. This may need overriding for multi-homed hosts. See `environments/common/inventory/group_vars/all/cloud_init.yml` for configuration.
+Hosts in the `etc_hosts` groups have `/etc/hosts` created with entries of the format `IP_address canonical_hostname [alias]`.
 
-# Variables:
+By default, an entry is created for each host in this group, using `ansible_host` as the IP_address and `inventory_hostname` as the canonical hostname. This may need overriding for multi-homed hosts or hosts with multiple aliases.
 
-- `etc_hosts_template`: Template file to use. Default uses in-role template.
+# Variables
+
+- `etc_hosts_template`: Template file to use. Default is the in-role template.
+- `etc_hosts_hostvars`: A list of variable names, used (in the order supplied) to create the entry for each host. Default is `['ansible_host', 'inventory_hostname']`
```

Comment:
which creates the following `/etc/hosts` line:

`<ip address> <ansible_host> <inventory_hostname>`

Reply:
resolved